pom.xml

```diff
@@ -164,12 +164,12 @@
         <plugin>
           <groupId>org.codehaus.mojo</groupId>
           <artifactId>animal-sniffer-maven-plugin</artifactId>
-          <version>1.14</version>
+          <version>1.22</version>
           <configuration>
             <signature>
               <groupId>org.codehaus.mojo.signature</groupId>
               <artifactId>java16</artifactId>
```

Comment:
Should be `java18`

By the way plugin looks like not used - only defined in `pluginManagement` 
no `executions`, no reference in `build/plugins`

Reply:
Yes, looks like it's not used, and according to the project:
> this plugin is now in maintenance level as the exact same feature can be now easily achieved using the --release flag from Javac

Removing the declaration...